README.md

```diff
@@ -1027,6 +1042,66 @@ service:
     - echo "here I can run custom commands, but no preparation needed for this repo"
 ```
 
+## Custom Linters
+Some people and organizations may choose to have custom made linters run as a part of golangci-lint. That functionality 
+is supported through go's plugin library.
+
+### Create a Copy of `golangci-lint` that Can Run with Plugins
+In order to use plugins, you'll need a golangci-lint executable that can run them. The normal version of this project 
+is built with the vendors option, which breaks plugins that have overlapping dependencies.
+
+1. Download the `i473` branch of https://github.com/dbraley/golangci-lint/tree/i473
```

Comment:
We're going to merge it. Let's fix description.

Reply:
Yup, sorry, wanted the documentation to be accurate during the review. Will fix now.